yarn-project/prover-client/src/proving_broker/config.ts

```diff
@@ -64,6 +66,11 @@ export const proverBrokerConfigMappings: ConfigMappingsType<ProverBrokerConfig>
     description: 'The maximum number of epochs to keep results for',
     ...numberConfigHelper(1),
   },
+  proverBrokerStoreMapSizeKB: {
+    env: 'PROVER_BROKER_STORE_MAP_SIZE_KB',
+    description: "The size of the prover broker's database. Will override the dataStoreMapSizeKB if set.",
+    ...numberConfigHelper(1024),
```

Comment:
Is this a default value? We shouldn't provide a default here, it should adopt the general data store size.

Reply:
right not sure how that got int there 😕 fixed